frontend/src/tests/instructor-permission-test.js

```diff
@@ -516,51 +469,254 @@ export function instructorsPermissionTests(api) {
 
     it("fetch applications", async () => {
         await switchToInstructorOnlyUser();
-        let resp = await apiGET(
+        const resp = await apiGET(
             `/instructor/sessions/${session.id}/applications`
         );
         expect(resp).toHaveStatus("success");
 
         await restoreDefaultUser();
     });
 
-    it("fetch Ddahs", async () => {
-        // If a user is not in Instructors table, it is not considered an instructor
-        // for the purpose of fetching DDAHs - so we create one
-        const newInstructor = {
-            first_name: "Jane",
-            last_name: "Smith",
-            email: "jane.smith@gmail.com",
-            utorid: instructorUser.utorid,
-        };
-        let resp = await apiPOST(`/admin/instructors`, newInstructor);
-        expect(resp).toHaveStatus("success");
+    describe("Ddah permissions", () => {
+        let position;
+        let instructor;
+        let assignment;
+        let foreignAssignment;
+
+        beforeAll(async () => {
+            await restoreDefaultUser();
+
+            // Get our instructor
+            let resp = await apiGET(`/admin/instructors`);
+            expect(resp).toHaveStatus("success");
+            instructor = resp.payload.find(
+                (instructor) => instructor.utorid === instructorUser.utorid
+            );
+            expect(instructor).toBeDefined();
```

Comment:
Isn't this instructor already in the seed data?

Reply:
It is, but without the id, I need the id to supply later!

Reply:
No...that's not how it works. Seed data should always be an exact duplicate of what you'd get if you hit the endpoint.

Reply:
Oh, are you saying I can force the id of seeded instructor? I thought the database would insert it with whatever id it wants?